sdk/python/lib/pulumi/metadata.py

```diff
@@ -27,3 +29,17 @@ def get_stack() -> str:
     Returns the current stack name.
     """
     return runtime.get_stack()
+
+
+def get_stack_tag(name: str) -> str:
+    """
+    Returns a stack tag's value or None.
```

Comment:
If this function can return `None`, its type should be `Optional[str]`

Reply:
Thanks! I'm a Python noob. Fixed.